cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1229,13 +1229,15 @@ function DownloadAndInstall-Crictl {
   if (-not (ShouldWrite-File ${env:NODE_DIR}\crictl.exe)) {
     return
   }
-  $url = ('https://storage.googleapis.com/kubernetes-release/crictl/' +
-      'crictl-' + $CRICTL_VERSION + '-windows-amd64.exe')
+  $CRI_TOOLS_GCS_BUCKET = "k8s-artifacts-cri-tools"
+  $url = ('https://storage.googleapis.com/$CRI_TOOLS_GCS_BUCKET/release/' +
```

Comment:
`$CRI_TOOLS_GCS_BUCKET` won't be substituted here because the string is in single-quotes. Either use double quotes here, or use + to construct.

Reply:
Please incorporate https://github.com/kubernetes/kubernetes/pull/91796.

If you have a GCP project you can then test this change with the same command I used there: `NUM_NODES=2 NUM_WINDOWS_NODES=2 KUBE_GCE_ENABLE_IP_ALIASES=true KUBERNETES_NODE_PLATFORM=windows LOGGING_STACKDRIVER_RESOURCE_TYPES=new KUBE_UP_AUTOMATIC_CLEANUP=true WINDOWS_NODE_OS_DISTRIBUTION=win2019 ./cluster/kube-up.sh`

Reply:
Thank you for the hint, changed as suggested. :+1: